docs/overview/versioning-policy.md

```diff
@@ -23,6 +23,10 @@ together. Think of it as a toolbox that comes with batteries included, but you
 can always add more plugins and libraries from the open source ecosystem as well
 as build your own.
 
+## Release Schedule
+
+Backstage releases occur on a monthly basis, specifically on the second Tuesday of each month. The first release took place in March 2022.
```

Comment:
hm don't releases go out on the third Tuesday? eg. A release came out today (7/18)
```suggestion
Backstage releases occur on a monthly basis, specifically on the third Tuesday of each month. The first release took place in March 2022.
```

Reply:
hehe yess, thanks for the heads up.